signin/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
+  <link rel="stylesheet" href="style.css">
+  <title>로그인</title>
+</head>
+<body>
+  <div class="login">
+    <header>
+      <div>
+        <a href="/"><img class="header-logo" src="/assets/image/logo.png"></a>
+      </div>
+      <div class="header-signup">
+        회원이 아니신가요? <a class="signup" href="/signup">회원 가입하기</a>
+      </div>
+    </header>
+    <section>
+      <div class="contents-frame">
+        <form class="contents-form">
+          <label for="email">이메일</label>
+          <input type="email" name="email" id="email" placeholder="이메일">
+          <label for="password">비밀번호</label>
+          <div class="password-container">
+            <div>
+              <input type="password" name="password" id="password" placeholder="비밀번호">
+            </div>
+            <div class="password-eye-container">
+              <img class="password-eye" src="/assets/image/eye-on.png">
+            </div>
+          </div>
+          <button type="submit">로그인</button>
+        </form>
+      </div>
+    </section>
+    <footer>
+      <div class="footer-frame">
+        <div class="footer-socialLogin-text">
+          소셜 로그인
+        </div>
+        <div class="footer-socialLogin-image">
```

Comment:
```suggestion
        <div class="footer-social-login-image">
```
네이밍 규칙 통일해주심 좋을 거 같아요!